bin/run-in-docker.sh

```diff
@@ -38,6 +38,7 @@ docker run \
     --network none \
     --mount type=bind,src="${solution_dir}",dst=/solution \
     --mount type=bind,src="${output_dir}",dst=/output \
+    --tmpfs /tmp:rw \
     exercism/test-runner "${slug}" "//solution" "//output"
```

Comment:
Should this be (looking at the mount):

```suggestion
    exercism/test-runner "${slug}" "/solution" "/output"
```

Reply:
yea, its kind of a mess, I'm running bash on windows, [which does magic things](https://newbedev.com/git-bash-string-parameter-with-at-start-is-being-expanded-to-a-file-path-how-to-stop-this)

anyhow, it now runs on github actions